source/install.rst

```diff
@@ -53,75 +53,105 @@ Note that ``e3sm_unified``'s development cycle is not in phase with ``e3sm_diags
 
```

Comment:
On line 10 can you also change `conda environment pulls together` to `conda environment that pulls together`?